VstsTfsSample/Program.cs

```diff
@@ -0,0 +1,122 @@
+﻿using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text.RegularExpressions;
+using Microsoft.VisualStudio.Services.Client;
+using Microsoft.VisualStudio.Services.WebApi;
+using Microsoft.TeamFoundation.Core.WebApi;
+using System.Collections.Generic;
+
+namespace VstsTfsSample
+{
+    class Program
+    {
+        //============= Config [Edit these with your settings] =====================
+        internal const string vstsOrTfsCollectionUrl = "https://myaccount.visualstudio.com"; //change to the URL of your VSTS or TFS account
```

Comment:
So you can't actually use ADAL with TFS that why the two variables with have different names.